README.md

```diff
@@ -1,4 +1,10 @@
-# wikit
+# WiKit
```

Comment:
We decided upon the wikit name already a while ago, but so far largely used it in the repo name, ... 
The way it is written here is probably how it is supposed to be in prose (based on https://github.com/wmde/wikit-css/) but I raised the question in the chat.

Happy to create a follow up to use it consistently (e.g. documentation) if that turns out positive.

Reply:
Will do the follow-up once #94 landed.